modulefiles/tasks/hera/get_obs_ccpa

```diff
@@ -0,0 +1,8 @@
+#%Module#####################################################
+## Module file for get_obs_ccpa task.
```

Comment:
Is it possible to have one modulefiles called get_obs that can be used for all three tasks? Similar to the run_vx modulefile that is used for the verification tasks?

Reply:
Good idea! I changed that.